### PR TITLE
Fixing minor issues in Sabre\DAV and its tests

### PR DESCRIPTION
**Pull request by jakobsack**
- remove unused variables
- fix function parameters
- fix some tests

### CODE REVIEW

**Comment by jakobsack** [comment]
It looks like I did not introduce any new failed errors.

File: tests/Sabre/DAV/Auth/Backend/PDOMySQLTest.php [hunk 1]
```diff
@@ -12,15 +12,17 @@ function getPDO() {
...
-	unique(username)
-);");
+  id integer unsigned not null primary key auto_increment,
+  username varchar(50),
+  digesta1 varchar(32),
+  email varchar(80),
+  displayname varchar(80),
+  unique(username)
+)
+SQL
```
Comment (staabm) [inline tests/Sabre/DAV/Auth/Backend/PDOMySQLTest.php]: missing semicolon?

Comment (jakobsack) [inline tests/Sabre/DAV/Auth/Backend/PDOMySQLTest.php]: Not really. Some calls have a semicolon, some don't. But as long as you fire one command at a time you don't need them. And crawling through the code I could not determine the "correct" way it is done in Sabre, it changes all the time.

Comment (staabm) [inline tests/Sabre/DAV/Auth/Backend/PDOMySQLTest.php]: I guess we need a semicolon for the heredoc?

Comment (jakobsack) [inline tests/Sabre/DAV/Auth/Backend/PDOMySQLTest.php]: Good point, I totally forgot about the semicolons in PHP. Too much ruby lately.

Comment (jakobsack) [inline tests/Sabre/DAV/Auth/Backend/PDOMySQLTest.php]: Sorry, the semicolon you are asking for is in line 25 ... The heredoc itself does not need a semicolon.

Comment (evert) [inline tests/Sabre/DAV/Auth/Backend/PDOMySQLTest.php]: :+1:

**Comment by evert** [comment]
Looks good to me. I'll look into the failing tests.

**Pull request merged**